packages/fxa-content-server/grunttasks/remarkable.js

```diff
@@ -2,48 +2,13 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-const path = require('path');
-const i18n = require('i18n-abide');
-
 module.exports = function (grunt) {
-  // convert localized TOS/PP agreements from markdown to html partials.
-
-  function rename(destPath, destFile) {
-    // Normalize the filenames to use the locale name.
-    // add the extension here, instead of using grunt-remarkable's
-    // extension generator, to get locale names in uppercase
-
-    const lang = i18n.localeFrom(destFile.split('/')[0]);
-    return path.join(destPath, lang + '.html');
-  }
-
   grunt.config('remarkable', {
```

Comment:
Are we still using this for something else or can this file be removed?

Reply:
As far as I can tell, we only used remarkable for TOS/PP stuffs, so we might be able to do some additional cleanup.

```sh
git grep -n "remarkable" packages | cat

packages/fxa-content-server/.nsprc:4:  "comment_745": "745 is RegExp denial by grunt-remarkable.",
packages/fxa-content-server/grunttasks/l10n-generate-tos-pp.js:14:        'remarkable:tos_pp',
packages/fxa-content-server/grunttasks/remarkable.js:6:  grunt.config('remarkable', {
packages/fxa-content-server/grunttasks/replace.js:21:          // such as `    [`, this creates a <code> block when parsed by remarkable
packages/fxa-content-server/package.json:101:    "grunt-remarkable": "1.1.0",
```

if https://github.com/mozilla/fxa/blob/main/packages/fxa-content-server/grunttasks/l10n-generate-tos-pp.js isn't used anymore, we might be able to start recursively removing more scripts and tasks:

https://github.com/mozilla/fxa/blob/658ddb7b31c237d397a201b10871b2c309bcef3e/packages/fxa-content-server/grunttasks/l10n-generate-tos-pp.js#L7-L18